<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectNormsDialog</class>
 <widget class="QDialog" name="ProjectNormsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Análise de Normas (Goiânia)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelExperimental">
     <property name="styleSheet">
      <string notr="true">background-color: yellow; color: black; font-weight: bold; padding: 2px;</string>
     </property>
     <property name="text">
      <string>ESTA FERRAMENTA ESTÁ EM FASE EXPERIMENTAL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBase">
     <property name="title">
      <string>Informações da Gleba (Área Total)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutBase">
      <item>
       <widget class="QLabel" name="labelBase">
        <property name="text">
         <string>Camada da Gleba Total:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsMapLayerComboBox" name="comboBaseLayer"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxNorms">
     <property name="title">
      <string>Normas de Parcelamento</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutNorms">
      <!-- We will define rows here. For now, 4 common categories -->
      <item>
       <layout class="QHBoxLayout" name="layoutRow1">
        <item>
         <widget class="QLabel" name="labelCat1">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Áreas Verdes:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsMapLayerComboBox" name="comboVerde"/>
        </item>
        <item>
         <widget class="QLabel" name="lblResultVerde">
          <property name="text">
           <string>0.00%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutRow2">
        <item>
         <widget class="QLabel" name="labelCat2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Institucionais:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsMapLayerComboBox" name="comboInst"/>
        </item>
        <item>
         <widget class="QLabel" name="lblResultInst">
          <property name="text">
           <string>0.00%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutRow3">
        <item>
         <widget class="QLabel" name="labelCat3">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Sistema Viário:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsMapLayerComboBox" name="comboViario"/>
        </item>
        <item>
         <widget class="QLabel" name="lblResultViario">
          <property name="text">
           <string>0.00%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutRow4">
        <item>
         <widget class="QLabel" name="labelCat4">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>APP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsMapLayerComboBox" name="comboAPP"/>
        </item>
        <item>
         <widget class="QLabel" name="lblResultAPP">
          <property name="text">
           <string>0.00%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutRow5">
        <item>
         <widget class="QLabel" name="labelCat5">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Reserva Legal:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsMapLayerComboBox" name="comboReserva"/>
        </item>
        <item>
         <widget class="QLabel" name="lblResultReserva">
          <property name="text">
           <string>0.00%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutRow6">
        <item>
         <widget class="QLabel" name="labelCat6">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Lotes/Quadras:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsMapLayerComboBox" name="comboLotes"/>
        </item>
        <item>
         <widget class="QLabel" name="lblResultLotes">
          <property name="text">
           <string>0.00%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textSummary">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnCalculate">
       <property name="text">
        <string>Calcular Normas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
